DOM_creation/lesson1/index.html

```diff
@@ -1,3 +1,4 @@
+<!-- Lesson1 -->
 <!DOCTYPE html>
 <html lang="en">
```

Comment:
This assignment uses Japanese, so it is recommended that you set the language to Japanese.
```suggestion
<html lang="ja">
```